tests/bluetooth/audio/ascs/src/main.c

```diff
@@ -65,11 +65,10 @@ static void ascs_test_suite_fixture_init(struct ascs_test_suite_fixture *fixture
 		CONFIG_BT_ASCS_MAX_ASE_SNK_COUNT,
 		CONFIG_BT_ASCS_MAX_ASE_SRC_COUNT
 	};
-	int err;
 
 	memset(fixture, 0, sizeof(*fixture));
 
-	err = bt_bap_unicast_server_register(&param);
+	(void)bt_bap_unicast_server_register(&param);
```

Comment:
This isn't the correct solution. If you want to change this in this PR (IMO should be a separate PR), then it should be something like `zassert_equal(bt_bap_unicast_server_register(&param), 0);
or `zassert_equal(err, 0);`

Reply:
I only made the change to pass CI, since this is currently failing on main I assumed it would be properly fixed for the release.

Reply:
What was the CI error? Unused variable?

Reply:
yes

Reply:
Thanks. I'm having a look at how to properly fix this, but it seems non-trivial. Will let you know when I have a PR

Reply:
Here's the non-trivial and proper fix: https://github.com/zephyrproject-rtos/zephyr/pull/81515